extensions/eclipse-che-theia-testing-service-ext/src/browser/plugin-handle-registry.ts

```diff
@@ -0,0 +1,98 @@
+/********************************************************************************
+ * Copyright (C) 2020 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ ********************************************************************************/
+
+import { injectable } from 'inversify';
+import { LanguageServerAction } from './languages-test-main';
+import { LanguagesExt } from '@theia/plugin-ext';
+import { Emitter } from '@theia/core/lib/common/event';
+
+/**
+ * This class keeps a registry of which plugins map to which handle
+ */
+
+export interface LanguageFeatureRegistration {
+    languagesExtImpl: LanguagesExt; // The languagesMain that registered this plugin in the registry
+    providerHandles: Map<LanguageServerAction, number>; // A map of language actions to their handle
+}
+
+export interface LanguagesExtHandle {
+    handle: number;
+    languagesExt: LanguagesExt;
+}
+
+export interface LanguageRegistrationEvent extends LanguagesExtHandle {
+    pluginID: string;
+    action: LanguageServerAction;
+}
+
+// tslint:disable
```

Comment:
Didn't we move to eslint recently? Also, how far down the file does that disablement reach?

Reply:
I think only theia uses eslint right now. I think che-theia still uses tslint if I'm not mistaken